src/components/Recommendation.js

```diff
@@ -100,11 +100,14 @@ function Recommendation({ todayFutureToggler }) {
               {t('recommendation.negative')}
             </Header>
             <List>
-              {recommendations.negative.map(r => (
-                <List.Item key={r}>
-                  {translate('treeType', r, i18n.language)}
-                </List.Item>
-              ))}
+              {recommendations.negative.map(
```

Comment:
Have you checked why "recommendations.negative" contains null values?

Reply:
I think its because we have assigned empty array for three i.e. negative in recommend.js (line number 64 ) in library.

Reply:
An empty array should not contain null values.

Reply:
its because whilst producing json file from postgres, empty values were replaced with null. Updating migration file seems to be a solution.